randovania/data/json_data/prime3.txt

```diff
@@ -3763,24 +4616,37 @@ Landing Site Bravo
 Asset id: 4408585131827753141
 > Dock to Lift Hub Access; Heals? False
   * Normal Door to Lift Hub Access/Dock to Landing Site Bravo
+  > Samus Ship
+      Trivial
+
+> Samus Ship; Heals? True
+  > Dock to Lift Hub Access
+      Trivial
 
 ----------------
 Security Air Lock
 Asset id: 5682718801820042955
 > Dock to Lift Hub; Heals? False
   * Normal Door to Lift Hub/Dock to Security Air Lock
   > Dock to Defense Access
-      X-Ray Visor
+      After Security Air Lock Gate
+  > Pickup (Missile Expansion)
+      Trivial
 
 > Dock to Defense Access; Heals? False
   * Normal Door to Defense Access/Dock to Security Air Lock
   > Dock to Lift Hub
+      After Security Air Lock Gate
+  > Event - Gate
       X-Ray Visor
-  > Pickup (Missile Expansion)
-      Trivial
 
 > Pickup (Missile Expansion); Heals? False
   * Pickup 86; Major Location? False
+  > Dock to Lift Hub
+      Trivial
+
+> Event - Gate; Heals? False
+  * Event Security Air Lock Gate
   > Dock to Defense Access
       Trivial
 
```

Comment:
For Command Vault (Below):

- Change Docks to Morph Ball Doors
- Rename "Dock" part of the names to "Morph Ball Door"
- Add Missile > 5 to seeker requirement